### ISSUE TITLE
Place and group plants marker

### ISSUE THREAD

**Issue by dobro929** [naturerobots/HSOS-SEP-PlantMap-2022#203]

**Comment by lucaslootalot** [comment]
@jarkenau @pbrozi  There are only 1 to 2 beds per garden, which to be honest looks a bit sad in the table and on the map. Is it possible to expand that to maybe 5 beds? The data can be the same, the coordinates would only have to be moved.

**Comment by jarkenau** [comment]
@lucaslootalot I could add all of those:
![image](https://user-images.githubusercontent.com/49768113/184243630-a4025d8b-d21f-4dbb-96ac-35aba0cc109b.png)

It should be the same data on different positions.

**Comment by lucaslootalot** [comment]
Could you then check that the beds all have the offset coordinates and add them to the company 1 garden 1, that would be great! 😄

**Comment by lucaslootalot** [comment]
@dobro929  What kind of marker do we want to use? They don't look so suitable. 😿 
![2022-08-12_00h35_45](https://user-images.githubusercontent.com/57572111/184253933-69cdeb54-fc43-4fec-bf23-e0c7f97b53a2.png)

**Comment by dobro929** [comment]
@lucaslootalot a green marker already exists in `assets/marker`😄 You can use the green one or the crops icon (sidebar), as billie does.

**Comment by lucaslootalot** [comment]
Both doesnt look good. 
![image](https://user-images.githubusercontent.com/57572111/184314008-27bc0118-026f-4e7d-99b4-601ced46af11.png)
![image](https://user-images.githubusercontent.com/57572111/184314038-34908414-6a39-4bf7-a5e2-88975d5f3029.png)

**Comment by jarkenau** [comment]
@lucaslootalot Do the coordinates of the plants actually line up with the bed? Or did you manually tweak them ? :grinning:

**Comment by lucaslootalot** [comment]
I have moved the image. 🧙🏻‍♂️

**Comment by lucaslootalot** [comment]
@Anafabula Can you extend the `plant_coords` array of the beds with the IDs of the plants?

**Comment by dobro929** [comment]
I have added a few beds to the gardens 👍🏼

**Comment by lucaslootalot** [comment]
@dobro929 works, ty 👍🏻 
@jarkenau So you don't have to work on it anymore, thank you too . 😄

**Comment by jarkenau** [comment]
@lucaslootalot like this? 

```
{
    "id": bed.id,
    "plant": plant_type,
    "variety": variety,
    "plants": request.build_absolute_uri(plants_url),
    "soil_humidity": soil_humidity,
    "harvest": f"{harvest} week",
    "yield": pyield,
    "health": phealth,
    "plant_coords": plant_coords,
    "mid_latitude: number,
    "mid_longitude: number
}
```

**Comment by lucaslootalot** [comment]
@jarkenau Yes and additionally please add the plant ID to the objects in `plant_coords`. 🙇🏻
